windows/Toolbox.iss

```diff
@@ -184,9 +184,21 @@ begin
 end;
 
 function NeedToInstallGit(): Boolean;
+var
+  GitRegKey, GitInstallPath: String;  
 begin
-	// TODO: Find a better way to see if Git is installed
-	Result := not DirExists('C:\Program Files\Git') or not FileExists('C:\Program Files\Git\git-bash.exe')
+  GitRegKey := 'GitForWindows';
+
+  if RegQueryStringValue(HKEY_LOCAL_MACHINE, 'Software\Wow6432Node\' + GitRegKey, 'InstallPath', GitInstallPath) and FileExists(GitInstallPath + '\git-bash.exe') then
```

Comment:
Great! Any way this function could return the git path (and optionally Null)? This could be used to detect where to point the `Docker Quickstart Terminal` to